File: programs-basic-java/src/test/java/com/local/numbertypes/StrongNumberTest.java
package com.local.numbertypes;

import static com.local.numbertypes.NumberType.STRONG_NUMBER;

import java.util.List;

import org.junit.jupiter.api.Test;

class StrongNumberTest extends AbstractNumberTest {

	@Test
	void testStrongNumbers() {
		testNumbersTypeByFile(STRONG_NUMBER, "strongnumbers.txt");
	}

	@Test
	void testStrongNumbersByList() {
		// Pass the order as 5
		testNumbersTypeByList(STRONG_NUMBER, List.of(1, 2, 145), 3);
	}
}

File: programs-basic-kotlin/src/test/kotlin/com/local/numbertypes/StrongNumberTest.kt
package com.local.numbertypes
import org.junit.jupiter.api.Test
class StrongNumberTest : AbstractNumberTest() {
	@Test
	fun testStrongNumbers() {
		testNumbersTypeByFile(com.local.numbertypes.NumberType.STRONG_NUMBER, "strongnumbers.txt")
	}

	@Test
	fun testStrongNumbersByList() {
		// Pass the order as 5
		testNumbersTypeByList(com.local.numbertypes.NumberType.STRONG_NUMBER, java.util.List.of<Int>(1, 2, 145), 3)
	}
}